egs2/mini_an4/ssl1/run.sh

```diff
@@ -14,7 +14,7 @@ n_clusters_iter1=10
 
 feature_iter0="mfcc"
 layer_iter0="0"
-feature_iter1="hubert"
+feature_iter1="espnet_hubert"
```

Comment:
what is the difference between hubert and espnet_hubert?

Reply:
(hubert is from fairseq (so that we can use their checkpoint as well) while espnet_hubert is the one that from our side)

Reply:
So, in the previous script, we support the HuBERT ESPnet version, named as `hubert` as well as the legacy Fairseq version, named as `fairseq`. Since the `perform_kmeans.sh` inside the running script now supports `s3prl` based feature (including hubert as well). In this regard, I think renaming it as `fairseq_hubert` and `espnet_hubert` is more clear.